src/locales/en.json

```diff
@@ -23,6 +23,7 @@
     "fontFamily": "Font family",
     "onlySelected": "Only selected",
     "withBackground": "With Background",
+    "addWatermark": "Add watermark",
```

Comment:
I'm not sure that everyone is familiar with the word watermark. I feel like it would be better to rename this to `Add "Made in Excalidraw"` or something similar.